x-pack/plugins/observability/server/services/slo/delete_slo.ts

```diff
@@ -19,6 +19,9 @@ export class DeleteSLO {
   ) {}
 
   public async execute(sloId: string): Promise<void> {
+    // ensure the slo exists on the request's space.
+    await this.repository.findById(sloId);
```

Comment:
💬  if the request is made from a space where the slo is not available, the repository throws a SLONotFound error.

Reply:
It makes senses as we are now relying on the API of Kibana